### ISSUE TITLE
Add support for ghc-9.8

### ISSUE THREAD

**Issue by erikd** [sdiehl/protolude#145]
Builds fine with the following patch:
```diff
diff --git a/protolude.cabal b/protolude.cabal
index c02dc8a..5705d64 100644
--- a/protolude.cabal
+++ b/protolude.cabal
@@ -69,15 +69,15 @@ library
       array                >=0.4  && <0.6
     , async                >=2.0  && <2.3
     , base                 >=4.6  && <4.20
-    , bytestring           >=0.10 && <0.12
+    , bytestring           >=0.10 && <0.13
     , containers           >=0.5  && <0.8
-    , deepseq              >=1.3  && <1.5
-    , ghc-prim             >=0.3  && <0.11
+    , deepseq              >=1.3  && <1.6
+    , ghc-prim             >=0.3  && <0.12
     , hashable             >=1.2  && <1.5
     , mtl                  >=2.1  && <2.4
     , mtl-compat           >=0.2  && <0.3
     , stm                  >=2.4  && <2.6
-    , text                 >=1.2  && <2.1
+    , text                 >=1.2  && <2.2
     , transformers         >=0.2  && <0.7
     , transformers-compat  >=0.4  && <0.8
 
```
but requires the following `cabal.project` file:
```
packages:
  .

allow-newer:
  , async:base
```

Let me know if you would prefer a PR.

**Comment by andreasabel** [comment]
I published revision 2 of 0.3.3 to allow building with GHC 9.8.

**Comment by erikd** [comment]
Protolude `3.4` has been released so this can be closed.
